### PR TITLE
Add cluster-roles for kubedbAutoscaler

### PR DESCRIPTION
**Pull request by ArnobKumarSaha**
Signed-off-by: Arnob kumar saha <arnob@appscode.com>

### CODE REVIEW

File: charts/kubedb-autoscaler/templates/cluster-role.yaml [hunk 1]
```diff
@@ -5,68 +5,87 @@ metadata:
...
+    resources:
+      - statefulsets/finalizers
+    verbs: ["update"]
+  - apiGroups:
+      - apps
+    resources:
+      - statefulsets
+      - deployments
+      - daemonsets
+      - replicasets
```
Comment (tamalsaha) [inline charts/kubedb-autoscaler/templates/cluster-role.yaml:47]: Why do we need others.

File: charts/kubedb-autoscaler/templates/cluster-role.yaml [hunk 2]
```diff
@@ -5,68 +5,87 @@ metadata:
...
+    verbs: ["get", "list", "watch"]
+  - apiGroups:
+      - batch
+    resources:
+      - jobs
+      - cronjobs
+    verbs: ["get", "list", "watch"]
```
Comment (tamalsaha) [inline charts/kubedb-autoscaler/templates/cluster-role.yaml:54]: Why do we need this?

File: charts/kubedb-autoscaler/templates/cluster-role.yaml [hunk 3]
```diff
@@ -5,68 +5,87 @@ metadata:
...
+  - apiGroups:
+      - ""
+    resources:
+      - events
+    verbs: ["get", "list", "watch", "create"]
+  - apiGroups:
+      - "metrics.k8s.io"
+    resources:
+      - pods
+    verbs: ["get", "list", "watch", "create"]
```
Comment (tamalsaha) [inline charts/kubedb-autoscaler/templates/cluster-role.yaml:91]: Remove unused verbs.

Comment (ArnobKumarSaha) [reply charts/kubedb-autoscaler/templates/cluster-role.yaml]: All the unnecessary stuffs removed

**Review by faem — APPROVED** [review]

**Pull request merged**